src/main/java/appeng/client/ClientHelper.java

```diff
@@ -85,30 +84,14 @@ public void preinit()
 		{
 			ModelLoaderRegistry.registerLoader( UVLModelLoader.INSTANCE );
 		}
+
+		RenderingRegistry.registerEntityRenderingHandler( EntityTinyTNTPrimed.class, manager -> new RenderTinyTNTPrimed( manager ) );
+		RenderingRegistry.registerEntityRenderingHandler( EntityFloatingItem.class, manager -> new RenderFloatingItem( manager ) );
 	}
 
 	@Override
 	public void init()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1186)